lib/fog/compute/ecloud/models/ssh_key.rb

```diff
@@ -9,6 +9,30 @@ class SshKey < Fog::Ecloud::Model
         attribute :other_links, :aliases => :Links
         attribute :default, :aliases => :Default, :type => :boolean
         attribute :finger_print, :aliases => :FingerPrint
+        attribute :private_key, :aliases => :PrivateKey
+
+        def delete
+          service.ssh_key_delete(href).body
+        end
+        alias_method :destroy, :delete
+
+        def edit(options = {})
```

Comment:
~~Is `edit` something the other models are using?~~

Just realized it is. Maybe we should not add an alias here since the other models don't?